app/views/records/_filter_tab.html.haml

```diff
@@ -8,10 +8,14 @@
   \# http://digitalnz.org/supplejack
 .content{:id => "panel1-#{index + 2}"}
   %section
-    %ul.columns
-      - Hash[facet_values.sort].each do |label, count|
-        %li.large-3.columns
+    - if facet[0] == 'usage'
```

Comment:
This will also need to be updated from the changes here https://github.com/DigitalNZ/supplejack_website/pull/22/files#r58474545